Source/WebInspectorUI/UserInterface/Views/TabBar.css

```diff
@@ -41,21 +41,20 @@
     --tab-bar-border-z-index: 10;
 }
 
-body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar {
+body.mac-platform .tab-bar {
     --tab-bar-background: hsl(0, 0%, 90%);
 
     --tab-item-light-border-color: hsl(0, 0%, 85%);
     --tab-item-medium-border-color: hsl(0, 0%, 85%);
     --tab-item-dark-border-color: hsl(0, 0%, 75%);
 }
 
-body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar {
+body:not(.mac-platform) .tab-bar {
     background-size: 100% 200%;
     --tab-bar-background: linear-gradient(to bottom, hsl(0, 0%, 78%), hsl(0, 0%, 72%));
 }
 
-body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar,
-body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar {
```

Comment:
This confusing selector was most likely written to win a specificity race (the arguments inside `:is()` and `:not()` [contribute to overall selector specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/:is#difference_between_is_and_where)).

The race is between `body.mac-platform.monterey` and `body.window-inactive` and the former wins. 
This actually caused a bug where the window-inactive styles were overwritten when Web Inspector was not focused.

After we simplify the style-guarded selectors down to `body.mac-platform` it's becomes even specificity with `body.window-inactive`. We rely on CSS order for the `.window-inactive` to apply.

**Before**
<img width="537" alt="undocked-area-inactive-before" src="https://github.com/user-attachments/assets/68e9a359-c052-41d6-86b4-f10246d8ffe9">

**After**
<img width="538" alt="undocked-area-inactive-after" src="https://github.com/user-attachments/assets/73faf814-a925-4a77-95d0-f63c69b03f59">

This fixes the issue and brings the unfocused undocked Web Inspector UI title bar in line with macOS design (from white to gray):

**Before**
<img width="1019" alt="light-inactive-before" src="https://github.com/user-attachments/assets/1646f48c-ba69-41f7-b451-a393eee61d79">

**After**
<img width="1019" alt="light-inactive-after" src="https://github.com/user-attachments/assets/7710ee0e-2e54-4719-a117-7ed446a20ad7">